content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,576 @@
+# canonical_url: http://somewhere.else/ # This will be added in future
+---
+
+![Image of a Greek god holding a torch with the Prometheus logo, and OTel logo](Prom-and-otel-logos.png)
+
+Tools such as [Prometheus](https://prometheus.io/) and [OpenTelemetry](/docs/what-is-opentelemetry/) 
+help us monitor the health, performance, and 
+availability of our complex distributed systems. Both are open source projects 
+under the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) 
+umbrella – but what role does each play in observability? 
+
+OpenTelemetry (OTel for short), is a vendor-neutral open standard for 
+instrumenting, generating, collecting, and exporting telemetry data.
+Prometheus is a fixture of the observability landscape, widely relied upon 
+for monitoring and alerting within organizations. 
+
+While both Prometheus and OTel emit metrics, there is a lot to cover on the 
+differences and similarities, and is out of scope for this article. Rather, we 
+want to show you how OTel supports Prometheus, specifically in a Kubernetes 
+environment. You'll learn:
+* How the OTel Collector's Prometheus Receiver can be used to ingest Prometheus 
+metrics. 
+* Alternative methods for Prometheus metric collection through OTel-native 
+options such as the K8s cluster receiver and Kubelet stats receiver.
+
+We'll also do a technical dive into the OTel Operator's Target Allocator (TA) 
+and learn:
+* How it can be used for Prometheus service discovery.
+* How it ensures even distribution of Prometheus targets. 
+
+## OTel vs Prometheus
+
+Since OTel is primarily focused on the instrumentation part of observability,
+it doesn't provide a backend for storing telemetry; you have to forward the 
+data to a backend vendor for storage, alerting, and querying. 
+
+Prometheus, on the other hand, provides a time-series data store you can use 
+for your metrics, in addition to instrumentation clients. You can view 
+graphs and charts, set up alerts, and query your data via their web user 
+interface. It also encompasses a data format, known as [Prometheus 
+text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats).
+
+Prometheus _data_ is stored as a dimensional time-series, meaning that the data 
+has attributes (for example, labels or dimensions) and a timestamp. 
+
+The Prometheus _server_ collects Prometheus metrics data from targets defined in a 
+configuration file. A target is an endpoint that supplies metrics for the 
+Prometheus server to store.
+
+Prometheus is so ubiquitous in the monitoring space that many tools natively 
+emit metrics in Prometheus format, including [Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/) and [HashiCorp's Nomad](https://developer.hashicorp.com/nomad/docs/operations/monitoring-nomad). And for those that don’t, there are a number 
+of vendor- and community-built Prometheus clients to aggregate and import data 
```

Comment:
```suggestion
of vendor- and community-built [Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/) to aggregate and import data 
```

Unfortunately the correct term here is a Prometheus exporter tbh. Prometheus client would mean the SDK.